VERSION

```diff
@@ -1 +1 @@
-1.2.0
+1.3.0-rc1
```

Comment:
Did we miss the 1.3.0-rc0 release on this one?

Reply:
Looks like we did , we are not bumping for stable releases only tagging, this repository should not break kata so we only maintain one branch and bump when kata in master has a new release.

Reply:
We could therefore add the `-rc0` tag at the correct commit now right?

Reply:
agreed @jodh-intel

Reply:
Thanks for adding it @jcvenegas!